middle-section.html

```diff
@@ -0,0 +1,121 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="middle-section.css">
+</head>
+<body>
+    <!-- this container is a grid container with a single column --> -->
+    <div class="works-section">
+    <section>
+    <header>
+        <h1 class="my-works-header"><img src="Title.png" alt=""></h1> 
+        <h2 class="my-works-header2"><img src="Vectorline.png" alt="" width="315"></h2> 
+        </header>
+        <!-- this container is a flex container containing all the cards used. -->
+    <section class="show-case">
+        <img src="Frame44.png" alt="">
+   <div class="project-list1">
+    <div class="p-image"></div>
+   </div>
+   <!-- //this container list all projects on the site -->
+   <div class="project-list">
+<section>
+<header class="P-Info1"><h1></h1></header>
+<p class="content-1"></p>
+<ul>
+    <li class="css"></li>
+    <li class="html"></li>
+    <li  class="bootstrap"></li>
+    <li  class="ruby"></li>
+</ul>
+<ul><li><button class="se"></button></li></ul>
+
+</section>
+   </div>
+   <div class="project-list2">
+    <section class="t-post">
+    
+        <header class="P-Info2"><h1 class="t-head2"></h1></header>
+<p class="content-2"></p>
+    <ul class="ul2">
+        <li class="html2"></li>
+        <li  class="bootstrap2"></li>
+        <li  class="ruby2"></li>
+    </ul>
+    
+    </section>
+    <ul><li><button class="se"></button></li></ul>
+   </div>
+    </section>
+<!-- /// this section is the about myself section containing all the information about the owner of the site and his achievenebts -->
+    <section class="About-us">
+      
+    <header>
+        <h1 class="about-us-header"><img src="aboutus.png" alt=""></h1> 
+        </header>
+        <div>
+        <p class="describe"><img src="Description.png" alt="" width="315"></p> 
+        <p class="Get-my-resume"><img src="Enable.png" alt=""></p>
+    </div>
+        <div class="white-image">
+            <img src="Frame.png" alt="">   
+        </div>
+    <section class="programme-list">
+    <section class="categories">
+        
+  <header>
+    <h1 class="cat-header"><img src="images/Rectangle55.png" alt=""> <img src="images/Language.png" alt=""></h1>
+  </header>
+  <ul class="category-items">
+  <li><img src="images/Javascript.png" alt=""></li>
+  <li><img src="images/Ruby.png" alt=""></li>
+  <li><img src="images/Html.png" alt=""></li>
+  <li><img src="images/Css.png" alt=""></li>
+    </ul>
+</section>
+
+<section class="categories">
+        
+    <header>
+      <h1 class="cat-header"><img src="images/square.png" alt=""> <img src="images/Frameworks.png" alt=""></h1>
```

Comment:
- [ ] I kindly suggest that you do not make use of images for all of your full website instead use HTML markup 
- [ ] I kindly suggest that you use HTML markup and css to create your website and not just paste an image on your webpage. Please apply changes to everywhere affected in your HTML file.